core/neat/mixins/_grid-push.scss

```diff
@@ -0,0 +1,35 @@
+@charset "UTF-8";
+/// Push or pull a Neat grid column.
+///
+/// @argument {number (unitless)} $push [1]
+///
+/// @argument {map} $grid [$neat-default-grid]
+///   The grid to be used to generate the column. By default, the global
+///   `$neat-default-grid` will be used.
+///
+/// @example scss
+///   .element {
+///     @include grid-push(3);
+///   }
+///
+/// @example css
+///   .element {
+///     margin-left: calc((100% - (260px)) * 0.25 + 80px);
+///   }
+
+@mixin grid-push($push: false, $grid: $neat-default-grid) {
+  $_grid-columns: map-get($grid, columns);
+  $_grid-gutter: map-get($grid, gutter);
+
+  @if $push {
```

Comment:
I’m not sure we need the `@if` statement within this mixin, at all.

Are we trying to account for if someone `@include push`, but doesn’t pass a value?

Maybe instead we default to `0`/no push if no value passed to `$push`, and we throw a warning. Because either the mixin doesn’t need to be there (not actually pushing anything) or we need a value >= 1.